test/Microsoft.AspNetCore.Hosting.Tests/WebHostBuilderTests.cs

```diff
@@ -147,6 +147,98 @@ public void UseCaptureStartupErrorsHonored()
         }
 
         [Fact]
+        public void CodeBasedSettingsCodeBasedOverride()
+        {
+            var hostBuilder = new WebHostBuilder()
+                .UseSetting(WebHostDefaults.ServerKey, "ServerA")
+                .UseSetting(WebHostDefaults.ServerKey, "ServerB")
+                .UseServer(new TestServer())
+                .UseStartup<StartupNoServices>();
+            
+            var host = (WebHost)hostBuilder.Build();
+
+            Assert.Equal("ServerB", host.ServerFactoryLocation);
```

Comment:
Not sure if that's relevant for the test. I just want to make sure that when we build the host, the correct configuration settings are parsed. Whether they are used or not doesn't really matter.